tests/components/tibber/test_services.py

```diff
@@ -119,15 +120,13 @@ def create_mock_hass():
     return mock_hass
 
 
-def remove_microseconds(dt):
-    """Remove microseconds from a datetime object."""
-    return dt.replace(microsecond=0)
-
-
-async def test_get_prices():
+async def test_get_prices(
+    freezer: FrozenDateTimeFactory,
+) -> None:
     """Test __get_prices with mock data."""
-    today = remove_microseconds(dt.datetime.now())
-    tomorrow = remove_microseconds(dt.datetime.now() + dt.timedelta(days=1))
+    today = dt.datetime.now()
+    freezer.move_to(today)
```

Comment:
Why do we move to `now`? That doesn't make a lot of sense imo

Reply:
yeah - we could also define a constant for a defined time and use it everywhere instead of `now` :thinking: